static/js/apps/eval_rig/constants.tsx

```diff
@@ -27,3 +28,8 @@ export const DC_QUESTION_COL = "dc_question";
 export const DC_RESPONSE_COL = "dc_response";
 export const LLM_STAT_COL = "llm_stat";
 export const DC_STAT_COL = "dc_stat";
+
+export const DC_QUESTION_FEEDBACK_COL = "dc_question_feedback";
+export const DC_RESPONSE_FEEDBACK_COL = "dc_response_feedback";
+export const LLM_STAT_FEEDBACK_COL = "llm_stat_feedback";
+export const DC_STAT_FEEDBACK_COL = "dc_stat_feedback";
```

Comment:
super nit: maybe construct it: `DC_QUESTION_COL + FB_SUFFIX`, etc.

Reply:
good idea